tests/test_lhe_reader.py

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env python
+# coding=utf-8
+
```

Comment:
These are never going to be run in a standalone manner, so these can get removed.

Reply:
Ok, normally I add them just to be sure ;-) ... I have removed them in [10b08ee](https://github.com/scikit-hep/pylhe/pull/29/commits/10b08ee2b5668aafbf1fbc79db22841478b09744)